### PR TITLE
Refactor: remove StellarBanner

### PR DESCRIPTION
**Pull request by JoshuaHungDinh**
Resolves : [GIVE-1089]

## Description
Two separate errors reports were submitted for the recent StellarSaleBanner. One due to the session variables and the users unique setup & another from the FormGrid shortcode. These errors caused site crashes for these users. Due to the possibility of it affecting other users and in preparation of the v3 form as default. The Banner should be removed for now. 
During the next banner another method to handle banner alterations will need to be considered.

Related initial task references: 
- [GIVE-1020: Stellar Sale 2024 banners cause error on Donations & Donor pages for WP Engine user](https://stellarwp.atlassian.net/browse/GIVE-1020)
- https://github.com/impress-org/givewp/pull/7454

## Affects
Banner

## Visuals
N/A

## Testing Instructions
N/A 

## Pre-review Checklist
-   [x] Acceptance criteria satisfied and marked in related issue
-   [ ] Relevant `@unreleased` tags included in DocBlocks
-   [ ] Includes unit tests
-   [ ] Reviewed by the designer (if follows a design)
-   [x] [Self Review](https://give.gitbook.io/development-manual/devops/github/code-reviews#self-review) of code and UX completed

[GIVE-1089]: https://stellarwp.atlassian.net/browse/GIVE-1089?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Promotions/ServiceProvider.php [hunk 1]
```diff
@@ -62,12 +62,6 @@ private function bootPluginUpsells()
             Hooks::addAction('admin_enqueue_scripts', SaleBanners::class, 'loadScripts');
```
Comment (jonwaldstein) [inline src/Promotions/ServiceProvider.php:62]: We should prevent scripts from loading too

**Review by jonwaldstein — CHANGES_REQUESTED** [review]

File: src/Promotions/ServiceProvider.php (hunk 1 above)
Comment (JoshuaHungDinh) [reply src/Promotions/ServiceProvider.php]: Resolved: 44dad57ef8e69e392f2e6818e3a408382f463d43

**Review by jonwaldstein — APPROVED** [review]
👍

**Review by rickalday — APPROVED** [review]
Passed manual QA tests.

**Pull request merged**